.clang-format

```diff
@@ -1,2 +1,2 @@
-BasedOnStyle: LLVM
+BasedOnStyle: Google
```

Comment:
Agreed that it's worth updating `.clang-format` to reflect the current arrangement.